src/integrationTest/resources/application-integration.yml

```diff
@@ -1,5 +1,9 @@
 spring:
-  flyway.locations: classpath:db/testmigration
+  flyway:
+    locations: classpath:db/testmigration
+    enabled: true
+  aop:
+    proxy-target-class: true
 
 aggregation-strategy-completion-size: 100
 aggregation-strategy-timeout: 2000
```

Comment:
can we remove aggregation related properties as they are not used anymore?

Reply:
removed